src/www/app/server/server.spec.ts

```diff
@@ -58,19 +54,19 @@ describe('OutlineServerRepository', () => {
       storage
     );
     const server0 = repo.getById('server-0');
-    expect(server0?.accessKey).toEqual(shadowsocksConfigToAccessKey(CONFIG_0));
+    expect(server0?.accessKey).toEqual(CONFIG_0);
     expect(server0?.name).toEqual(CONFIG_0.name);
     const server1 = repo.getById('server-1');
-    expect(server1?.accessKey).toEqual(shadowsocksConfigToAccessKey(CONFIG_1));
+    expect(server1?.accessKey).toEqual(CONFIG_1);
     expect(server1?.name).toEqual(CONFIG_1.name);
   });
 
   it('loads V1 servers', () => {
     // Store V0 servers with different ids.
     const storageV0: ServersStorageV0 = {'v0-server-0': CONFIG_0, 'v0-server-1': CONFIG_1};
     const storageV1: ServersStorageV1 = [
-      {id: 'server-0', name: 'fake server 0', accessKey: shadowsocksConfigToAccessKey(CONFIG_0)},
-      {id: 'server-1', name: 'renamed server', accessKey: shadowsocksConfigToAccessKey(CONFIG_1)},
+      {id: 'server-0', name: 'fake server 0', accessKey: CONFIG_0.toString()},
+      {id: 'server-1', name: 'renamed server', accessKey: CONFIG_1.toString()},
```

Comment:
I'm pretty sure I broke how the name-override thing works, note to self for this.

Reply:
I'm also going to move the unwrap logic into access_config.ts i guess